AdvisoryNotifications/.repo-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "distribution_name": "google/cloud-advisorynotifications",
+    "release_level": "preview",
+    "client_documentation": "https://cloud.google.com/php/docs/reference/cloud-advisorynotifications/latest",
+    "library_type": "GAPIC_AUTO",
+    "api_shortname": "advisorynotifications"
+}
```

Comment:
nit, do we need a new line at EoF here? IDK if its that big of a deal

Reply:
Done!